manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
+InstallerType: inno
+FileExtensions:
+- sz
+AppsAndFeaturesEntries:
+- DisplayName: SzArchiver 0.84
+  Publisher: Super Flexible Software
+  DisplayVersion: 0.84
+  ProductCode: SzArchiver_is1
```

Comment:
```suggestion
```
Not needed for this package as the normalization functions of winget will allow for a match using the PackageName, Publisher, and PackageVersion. Additionally, the DisplayVersion field should not be included if it matches the PackageVersion as it causes additional mapping on the backend which is undesirable.